pkg/integrations/v4/supervisor_fb_conf_linux_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//go:build linux
+
+package v4
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_defaultLoggingBinDir(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name           string
+		ffEnabled      bool
+		ffExists       bool
+		expectedBinDir string
+	}{
+		{
+			name:           "no ff",
+			ffExists:       false,
```

Comment:
For visibility maybe we can add `ffEnabled: false,` 🤔

Reply:
changed in c732cd1e33490d7a1bbd08640394eb9533913393